extra/debian/control

```diff
@@ -59,7 +59,6 @@ Depends: ${shlibs:Depends},
          gstreamer1.0-plugins-base,
          gstreamer1.0-plugins-good,
          gstreamer1.0-tools,
-         gstreamer1.0-x,
```

Comment:
Good point about xvimagesink, it hadn't occurred to me.  We need to keep this considering our default sink is xvimagesink.